### PR TITLE
ELPP-3163 Proofreader upgrade to 0.2

### PR DESCRIPTION
**Pull request by giorgiosironi**
How https://github.com/elifesciences/proofreader-php/pull/7 can be used (depends on that of course, but can be built locally by rebuilding the `elifesciences/proofreader-php` image).

Supports this complex command to run the proofreader image on the host's files:
```
docker run -v $(pwd):/code -v $(pwd)/.php_cs.cache:/srv/proofreader-php/.php_cs.cache -u $(id -u) elifesciences/proofreader-php vendor/bin/php-cs-fixer fix /code/src
```

This will conflict with other PRs, so it's a proof of concept, not meant to be merged.

### CODE REVIEW

File: Dockerfile.ci [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ARG commit=develop
-FROM elifesciences/proofreader-php:0.1 AS proofreader
+FROM elifesciences/proofreader-php:0.2 AS proofreader
 FROM elifesciences/annotations_cli:${commit}
 ENV DEBUG=1
 
 USER root
-RUN mkdir -p build && chown www-data:www-data build
+RUN mkdir -p build && chown www-data:www-data build && \
+    touch .php_cs.cache && chown www-data:www-data .php_cs.cache
```
Comment (giorgiosironi) [inline Dockerfile.ci]: this is a default, but it can be overridden with a volume if needed

Comment (giorgiosironi) [reply Dockerfile.ci]: (so I'll separate that in a follow-up PR as there may be some decisions to take like where to put it on the host and in the container)

**Comment by giorgiosironi** [comment]
Ready for a new review, this time it's green.

File: .php_cs [hunk 2]
```diff
@@ -1,12 +1,16 @@
 <?php
 
-$finder = Symfony\CS\Finder\DefaultFinder::create()
+$finder = PhpCsFixer\Finder::create()
     ->exclude('var')
-    ->name('console')
```
Comment (thewilkybarkid) [inline .php_cs]: Means it no longer checks the `bin/console` file?

Comment (giorgiosironi) [reply .php_cs]: The finder find it by adding `name()`, but the fixer ignores it nonetheless. Investigating.

File: .php_cs [hunk 3]
```diff
@@ -1,12 +1,17 @@
 <?php
 
-$finder = Symfony\CS\Finder\DefaultFinder::create()
+$finder = PhpCsFixer\Finder::create()
     ->exclude('var')
     ->name('console')
```
Comment (giorgiosironi) [inline .php_cs]: tested `console` and it's now working

**Pull request merged**